### ISSUE TITLE
Cannot compile project on fresh clone

### ISSUE THREAD

**Issue by MZochowskiSUMO** [SumoLogic/sumologic-log4j2-appender#39]
When running `mvn clean package` on a fresh clone I get the following error:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project sumologic-log4j2-appender: Compilation failure: Compilation failure: 
[ERROR] Source option 6 is no longer supported. Use 7 or later.
[ERROR] Target option 6 is no longer supported. Use 7 or later.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

**Comment by mccartney** [comment]
I bet you are using a fairly new Java. I bet using JRE 8 will work for you.

Anyway, Java 6 is archaic, there's no reason for us to support it anymore.

**Comment by mccartney** [comment]
Released the version `2.0` which requires Java 8 now.

**Comment by MZochowskiSUMO** [comment]
The release of new version solved the problem. Thanks!
